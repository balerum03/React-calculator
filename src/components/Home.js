import React from 'react';

const Home = () => (
  <div className="home">
    <h1>Welcome To my Calculator</h1>
    <p>On this page you will find an online calculator that was build with React</p>
  </div>
);

export default Home;
