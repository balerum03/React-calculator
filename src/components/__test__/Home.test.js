import react from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

test('Component renders correctly', () => {
  const component = renderer.create(
    <Home />,
  )
  .toJSON();
  expect(component).toMatchSnapshot();
});