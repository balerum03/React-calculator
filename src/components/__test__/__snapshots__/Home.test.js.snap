// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<div
  className="home"
>
  <h1>
    Welcome To my Calculator
  </h1>
  <p>
    On this page you will find an online calculator that was build with React
  </p>
</div>
`;
